18F/identity-idp
# Convert foundation-emails to Sass module syntax

## 🛠 Summary of changes

Replaces the last of Sass `@import` with equivalent `@use` and `@forward` [Sass module syntax](https://github.com/sass/sass/blob/main/accepted/module-system.md).

**Why?**

- [`@import` is deprecated](https://sass-lang.com/documentation/at-rules/import)

Previously: https://github.com/18F/identity-idp/pull/8651#issuecomment-1607420448

## 📜 Testing Plan

1. `yarn build:css`
2. `rails s`
3. Visit http://localhost:3000/rails/mailers/user_mailer/email_confirmation_instructions
4. Observe that the email preview looks the same as it would on `main`
